components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -219,6 +219,13 @@ class NewTabPage extends React.Component<Props, State> {
     }
 
     this.props.saveShowBinance(!showBinance)
+
+    // If we are about to hide the widget, disconnect
+    if (showBinance) {
```

Comment:
Note for reviewer: This doesn't need to be `!showBinance` as this value represents the current state of the visibility, this is a toggle function that passes in the opposite.